pkg/gpioControl/gpioControlChardev.go

```diff
@@ -0,0 +1,170 @@
+package gpioControl
+
+import (
+	"3mdeb/RteCtrl/pkg/config"
+	"fmt"
+
+	"github.com/warthog618/go-gpiocdev"
+)
+
+type GpioChardev struct {
+	sysGpioPath string
+	gpios       map[int]pin
+}
+
+func (ctrl *GpioChardev) GetPinNumByID(id int) int {
+	return int(ctrl.gpios[id].sysNum)
+}
+
+func RequestLine(sysGpio int, options ...gpiocdev.LineReqOption) (*gpiocdev.Line, error) {
+  var line *gpiocdev.Line
+  var err error
+
+  fmt.Printf("Initializing line %d...\n", sysGpio)
+  if sysGpio >= 400 {
+    line, err = gpiocdev.RequestLine("gpiochip2", sysGpio - 400, options...)
```

Comment:
What are these chips?
One is from GPIO expander IIRC? Can we use defines and name them accordingly?